apps/qubit/modules/informationobject/templates/_cardViewResults.php

```diff
@@ -11,15 +11,15 @@
     <?php endif; ?>
 
       <a href="<?php echo url_for(array('module' => 'informationobject', 'slug' => $doc['slug'])) ?>">
-        <?php if (!empty($doc['digitalObject']['thumbnailPath'])
+        <?php if (isset($doc['digitalObject']) && !empty($doc['digitalObject']['thumbnailPath'])
```

Comment:
Are you sure this change is necessary? empty() not enough? This seems to work for me:

```php
<?php

error_reporting(E_ALL);
ini_set('display_errors', '1');

$a = array();
empty($a['foo']['bar']);